NEWS.md

```diff
@@ -14,6 +14,8 @@
 
 1. `by=.EACHI` when `i` is keyed but `on=` different columns than `i`'s key could create an invalidly keyed result, [#4603](https://github.com/Rdatatable/data.table/issues/4603) [#4911](https://github.com/Rdatatable/data.table/issues/4911). Thanks to @myoung3 and @adamaltmejd for reporting, and @ColeMiller1 for the PR. An invalid key is where a `data.table` is marked as sorted by the key columns but the data is not sorted by those columns, leading to incorrect results from subsequent queries.
 
+2. `as.data.table()` more robust to unrecognized subclasses of `data.frame`, [#4526](https://github.com/Rdatatable/data.table/issues/4526). Thanks @RicoDiel for the report.
```

Comment:
A little more detail in the news item please:
i) what was not robust before; e.g. was it an error or warning before, or was the result wrong?
ii) what is an example of an 'unrecognized subclass'?

Reply:
had another go at it, LMK WYT

Reply:
@MichaelChirico I don't see any changes. Did you commit and push?

Reply:
i must have forgotten to push, done now

Reply:
strange, I still don't see the news item updated.

Reply:
Hmm I am also seeing it rendered the same in this comment thread. But on the `Changes` page it's different. I guess a thread is anchored to a commit in chase the anchoring spot is moves or disappears. Marking this as resolved to hopefully remove the confusion.

Reply:
I had been looking at the files-changed tab, and the changes button, but I was expecting a bigger change to the news item so my eyes were seeing the first 'more robust to subclasses' and thinking it wasn't changed. Then the several commits were labelled merges which threw me off.   Ok, on the same page now.

The idea is to be very specific in news items, e.g. including an example error message so it comes up in when folk search for parts of the error message.  I'll have a go...

Reply:
Done. I added your name too given #4959 and I'll add a comment there too.